src/components/Blog/BlogArticlesCollections/Pagination.style.ts

```diff
@@ -0,0 +1,78 @@
+import type { IconButtonProps } from '@mui/material';
+import { Box, IconButton } from '@mui/material';
+
+import { getContrastAlphaColor } from '@/utils/colors';
+import { styled } from '@mui/material/styles';
+
+export const PaginationContainer = styled(Box)(({ theme }) => ({
```

Comment:
Duplication of the BlogArticlesBoard

Reply:
Yes, cleaned up that junk and have a single component instead now! Thanks